pre-commit

```diff
@@ -7,7 +7,7 @@ RC=0
 subhook_root=$git_root/.git/hooks/commit_hooks
 
 hook_dir="$(dirname $0)"
-hook_symlink="$(readlink $0)"
+hook_symlink="$(readlink -f $0)"
```

Comment:
I just realized that mac's default readlink doesn't have a -f option, I use GNU's readlink on my mac which is why I hadn't noticed at first, sorry about that.